test/images/serve_hostname/Makefile

```diff
@@ -31,7 +31,7 @@ ALL_ARCH = amd64 arm arm64 ppc64le s390x
 
 GOARM=7
 TEMP_DIR := $(shell mktemp -d)
-GOLANG_VERSION=1.7.5
+GOLANG_VERSION=1.8
```

Comment:
We don't want to use latest - we definitely want to pin it to a specific version.